protected/humhub/libs/DynamicConfig.php

```diff
@@ -91,37 +90,10 @@ public static function rewrite()
         $config['name'] = Yii::$app->settings->get('name');
 
         // Add Caching
-        $cacheClass = Yii::$app->settings->get('cache.class');
-        if (in_array($cacheClass, ['yii\caching\DummyCache', 'yii\caching\FileCache'])) {
-            $config['components']['cache'] = [
-                'class' => $cacheClass,
-                'keyPrefix' => Yii::$app->id,
-            ];
-        } elseif ($cacheClass == 'yii\caching\ApcCache' && (function_exists('apcu_add') || function_exists('apc_add'))) {
-            $config['components']['cache'] = [
-                'class' => $cacheClass,
-                'keyPrefix' => Yii::$app->id,
-                'useApcu' => (function_exists('apcu_add')),
-            ];
-        } elseif ($cacheClass === \yii\redis\Cache::class) {
-            $config['components']['cache'] = [
-                'class' => \yii\redis\Cache::class,
-                'keyPrefix' => Yii::$app->id,
-            ];
-        }
-
-        // Add User settings
-        $config['components']['user'] = [];
-        if (Yii::$app->getModule('user')->settings->get('auth.defaultUserIdleTimeoutSec')) {
-            $config['components']['user']['authTimeout'] = Yii::$app->getModule('user')->settings->get('auth.defaultUserIdleTimeoutSec');
-        }
 
-        // Install Mail Component
-        $config['components']['mailer'] = self::getMailerConfig();
 
         // Remove old theme/view stuff
         unset($config['components']['view']);
-        unset($config['components']['mailer']['view']);
```

Comment:
We should `unset` all old `dynamic.php` vars, otherwise it would not be removed on the next save.

Reply:
this was done in second PR